src/worker.config.json

```diff
@@ -4,8 +4,8 @@
         "extensions":[".ps1", ".psm1"],
         "defaultExecutablePath":"dotnet",
         "defaultWorkerPath":"%FUNCTIONS_WORKER_RUNTIME_VERSION%/Microsoft.Azure.Functions.PowerShellWorker.dll",
-        "supportedRuntimeVersions":["6", "7"],
-        "defaultRuntimeVersion":"6",
+        "supportedRuntimeVersions":["7", "7.2"],
+        "defaultRuntimeVersion": null,
         "sanitizeRuntimeVersionRegex":"\\d+"
```

Comment:
We need to update this regex, otherwise "7.2" will be interpreted as "7". We need a regex that would map:

- `7` to `7`
- `~7` to `7`
- `7.2` to `7.2`

I think this will work: `"\\d+\\.?\\d*"`

Reply:
Good catch. Fixed.